[DOCU-849] 17.7 System admin design changes  https://community.openpr… (#24393)
opf/openproject: docs/system-admin-guide/design/README.md

![Branding settings in OpenProject administration](openproject_system_guide_design_branding.png)

![More branding settings in OpenProject administration](openproject_system_guide_design_branding_additional.png)

> Under **Default colors** you can configure a set of predefined colors which you can use e.g. to [set colors for work package types](https://www.openproject.org/docs/system-admin-guide/manage-work-packages/work-package-types) or for attribute highlighting, e.g. for [status](https://www.openproject.org/docs/system-admin-guide/manage-work-packages/work-package-status).

Reviewer: ```suggestion
Under **Default colors** you can configure a set of predefined colors which you can use e.g. to [set colors for work package types](../manage-work-packages/work-package-types) or for attribute highlighting, e.g. for [status](../manage-work-packages/work-package-status).
```

> Under **Default colors** tab you can configure a set of predefined colors in OpenProject which you can choose for e.g. [set colors for work package types](https://www.openproject.org/docs/system-admin-guide/manage-work-packages/work-package-types) or attribute highlighting, e.g. for [status](https://www.openproject.org/docs/system-admin-guide/manage-work-packages/work-package-status).

Reviewer 2: Nitpick: I stumbled over that sentence a bit..

```suggestion
Under **Default colors** you can configure a set of predefined colors which you can use e.g. to [set colors for work package types](https://www.openproject.org/docs/system-admin-guide/manage-work-packages/work-package-types) or for attribute highlighting, e.g. for [status](https://www.openproject.org/docs/system-admin-guide/manage-work-packages/work-package-status).
```
